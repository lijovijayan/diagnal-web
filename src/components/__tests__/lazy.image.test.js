import React from 'react'
import { shallow } from 'enzyme'
import toJson from 'enzyme-to-json'
import { LazyImage } from '../lazy.image'

describe('testing lazy image', () => {
    it('rendered properly', () => {
        const component = shallow(
            <LazyImage className={''} src={''} alt={''} />
        )
        const componentTree = toJson(component)
        expect(componentTree).toMatchSnapshot()
    })
})
