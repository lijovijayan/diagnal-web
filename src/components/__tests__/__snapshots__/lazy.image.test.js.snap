// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`testing lazy image rendered properly 1`] = `
<img
  alt=""
  className=" lazy-image"
  draggable={false}
  onError={[Function]}
  onLoad={[Function]}
  src="/images/placeholder_for_missing_posters.png"
/>
`;
